_submissions/round-12/0/2015-02-05-about-simon-fraser.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2015-02-05
+round: Round 12
+title: About Simon Fraser
+author: Simon Fraser
+permalink: /2015/02/simon-fraser/
+tags:
+  - Biography
+---
+Simon Fraser is Systems Administrator at the Wellcome Trust Sanger Institute,
+a genomics research charity near Cambridge, UK and the site that did a
+large portion of the work on the Human Genome Project. Simon writes 
+software for HPC operations in python, and is learning R, Go and Ruby.
+He also helps run the International Discworld Convention. 
```

Comment:
I suspect that we might have a higher than random sampling of Pterry fans here....